packages/riipen-ui/src/components/AsButton.test.jsx

```diff
@@ -0,0 +1,133 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import AsButton from "./AsButton";
+
+describe("AsButton component", () => {
+  const child = <div>A child</div>;
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<AsButton ariaLabel="click to be cool">{child}</AsButton>);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(
+      <AsButton ariaLabel="click to be cool">{child}</AsButton>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(
+        <AsButton ariaLabel="click to be cool">{child}</AsButton>
+      );
+
+      expect(wrapper.find("AsButton").props().classes).toEqual([]);
+      expect(wrapper.find("AsButton").props().clearButtonStyle).toEqual(true);
+      expect(wrapper.find("AsButton").props().type).toEqual("button");
+    });
+  });
+
+  describe("children prop", () => {
+    it("displays given children", () => {
+      const wrapper = mount(
+        <AsButton ariaLabel="click to be cool">{child}</AsButton>
+      );
+
+      expect(wrapper.containsMatchingElement(child)).toBeTruthy();
+    });
+  });
+
+  describe("classes prop", () => {
+    it("sets custom class", () => {
+      const classVariant = ["newClass"];
+
+      const wrapper = mount(
+        <AsButton ariaLabel="click to be cool" classes={classVariant}>
+          {child}
+        </AsButton>
+      );
+
+      expect(
+        wrapper
+          .find("AsButton")
+          .childAt(0)
+          .hasClass(classVariant[0])
+      ).toEqual(true);
+    });
+  });
+
+  describe("clearButtonStyle prop", () => {
+    it("Does not apply clearButtonStyle class when clearButtonStyle prop is false", () => {
```

Comment:
super duper nitpicky but should 'Does' be 'does' (lowercase)? Not sure what conventions are/if it matters capitalization/not for test description